translate/src/utils/message/getSimplePreview.test.js

```diff
@@ -35,6 +35,12 @@ describe('getSimplePreview', () => {
     expect(res).toEqual('Avengers');
   });
 
+  it('returns an empty string for an empty literal value', () => {
+    const message = 'empty = { "" }';
+    const res = getSimplePreview(message);
+    expect(res).toEqual('');
+  });
+
```

Comment:
Seems like this test cases also passes on master?

Reply:
Yeah. Fixed now to break there, once I figured out how the `StringLiteral` was sneaking in. Not entirely sure exactly how that's happening, but I'd rather not spend time fixing it, as this entire whole mode should disappear with the data model changes.